LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
😮

Did they require to add a License?

Reply:
Hmm, I don't think so but the projects we did in CodePath were all supposedly under Apache license so i I just followed it LOL Should i remove?

Reply:
It is fine, keep it. I was just curious.